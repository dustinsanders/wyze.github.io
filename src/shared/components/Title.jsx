// @flow

import type { Style, VNode } from '../../types'
import { css } from 'glamor'
import H1 from './H1'
import makeRGBA from '../helpers/makeRGBA'
import preact from 'preact'

type Props = {
  title: string,
}

const h1: Style = css({
  boxShadow: `0 2px 2px -2px ${makeRGBA(0.25)}`,
  color: makeRGBA(0.75),
  fontWeight: 300,
  marginBottom: '.5em',
  paddingBottom: '.25em',
  title: 'title',
})

const Title = ({ title }: Props): VNode => (
  <H1 styles={h1}>{title}</H1>
)

export default Title
