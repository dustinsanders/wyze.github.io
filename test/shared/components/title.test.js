// @flow

import Title from '../../../src/shared/components/Title'
import snapshot from '../../helpers/snapshot'

const title = 'Hello'

describe('<Title />', () => {
  it('renders', snapshot.bind(null, Title, { title }))
})
