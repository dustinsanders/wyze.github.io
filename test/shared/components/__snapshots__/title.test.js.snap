// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> renders 1`] = `"<H1 styles=\\"css-okjhmn\\">Hello</H1>"`;
